src/internal/testutil/pachctl.go

```diff
@@ -194,3 +195,11 @@ func (p *Pachctl) CommandTemplate(ctx context.Context, scriptTemplate string, da
 	}
 	return newCmd(ctx, "/bin/bash", nil, r), nil
 }
+
+// RunCommand runs command in sh (rather than bash), returning the combined
+// stdout & stderr.
+func (p Pachctl) RunCommand(ctx context.Context, command string) (string, error) {
+	cmd := exec.CommandContext(ctx, "/bin/sh", "-c", fmt.Sprintf(`PACH_CONFIG="%s" %s`, shellescape.Quote(p.configPath), command))
+	b, err := cmd.CombinedOutput()
```

Comment:
is there any existing client code that exercises a non-nil error here?

Reply:
There wasn’t, but I’ve added a bit to `TestSetProjectDefaults`. It’s a good thing to test, too — that invalid input generates a proper error.